### PR TITLE
cloud-sql-proxy: Add version 1.16

### PR DESCRIPTION
**Pull request by klaidliadon**

### CODE REVIEW

File: bucket/cloud-sql-proxy.json [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+            "url": "https://dl.google.com/cloudsql/cloud_sql_proxy_x86.exe#/cloud_sql_proxy.exe",
+            "hash": "d1576ef8c276b07ac970d2a1a0ce8db436863c1b7d5b83c8b290af8a7d28f456",
+            "bin": "cloud_sql_proxy.exe"
+        },
+        "64bit": {
+            "url": "https://dl.google.com/cloudsql/cloud_sql_proxy_x64.exe#/cloud_sql_proxy.exe",
+            "hash": "341a3f527e13a88758413b5f81391b18bf1d9268b5e52689815c726b8058b64b",
+            "bin": "cloud_sql_proxy.exe"
+        }
+    }
```
Comment (Ash258) [inline bucket/cloud-sql-proxy.json]: Hashes provided on github tags are mismatched so I would rather not extract it.

```suggestion
    "version": "1.16",
    "description": "Provides secure access to Cloud SQL Second Generation instances without having to add Authorized networks or configure SSL.",
    "homepage": "https://cloud.google.com/sql/docs/mysql/sql-proxy",
    "license": "Apache-2.0",
    "architecture": {
        "64bit": {
            "url": "https://storage.googleapis.com/cloudsql-proxy/v1.16/cloud_sql_proxy_x64.exe#/cloud_sql_proxy.exe ",
            "hash": "341a3f527e13a88758413b5f81391b18bf1d9268b5e52689815c726b8058b64b"
        },
        "32bit": {
            "url": "https://storage.googleapis.com/cloudsql-proxy/v1.16/cloud_sql_proxy_x86.exe#/cloud_sql_proxy.exe",
            "hash": "d1576ef8c276b07ac970d2a1a0ce8db436863c1b7d5b83c8b290af8a7d28f456"
        }
    },
    "bin": "cloud_sql_proxy.exe",
    "checkver": {
        "github": "https://github.com/GoogleCloudPlatform/cloudsql-proxy"
    },
    "autoupdate": {
        "architecture": {
            "64bit": {
                "url": "https://storage.googleapis.com/cloudsql-proxy/v$version/cloud_sql_proxy_x64.exe#/cloud_sql_proxy.exe "
            },
            "32bit": {
                "url": "https://storage.googleapis.com/cloudsql-proxy/v$version/cloud_sql_proxy_x86.exe#/cloud_sql_proxy.exe"
            }
        }
    }
```

**Comment by klaidliadon** [comment]
> gcloud is required?
> 
> image

Yes

File: bucket/cloud-sql-proxy.json [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+        "32bit": {
+            "url": "https://storage.googleapis.com/cloudsql-proxy/v1.16/cloud_sql_proxy_x86.exe#/cloud_sql_proxy.exe",
+            "hash": "d1576ef8c276b07ac970d2a1a0ce8db436863c1b7d5b83c8b290af8a7d28f456"
+        }
+    },
+    "bin": "cloud_sql_proxy.exe",
+    "checkver": {
+        "github": "https://github.com/GoogleCloudPlatform/cloudsql-proxy"
+    },
+    "depends": ["gcloud"],
```
Comment (Ash258) [inline bucket/cloud-sql-proxy.json]:
```suggestion
```

File: bucket/cloud-sql-proxy.json [hunk 3]
```diff
@@ -0,0 +1,31 @@
+{
+    "version": "1.16",
+    "description": "Provides secure access to Cloud SQL Second Generation instances without having to add Authorized networks or configure SSL.",
+    "homepage": "https://cloud.google.com/sql/docs/mysql/sql-proxy",
+    "license": "Apache-2.0",
+    "architecture": {
```
Comment (Ash258) [inline bucket/cloud-sql-proxy.json]:
```suggestion
    "depends": "gcloud",
    "architecture": {
```

**Pull request merged**